demo/web/index.html

```diff
@@ -3,11 +3,16 @@
   <head>
     <meta charset="utf-8">
     <title></title>
-    <script src="https://unpkg.com/react/umd/react.development.js"></script>
-    <script src="https://unpkg.com/react-dom/umd/react-dom.development.js"></script>
-    <script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/axios@0.18.0/dist/axios.min.js"></script>
-    <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css"></link>
+    <script 
+src="https://unpkg.com/react/umd/react.development.js"></script>
```

Comment:
Can we please undo the formatting changes in the PR, or at least fix the formatting, such that multi-line tags / script lines contain proper indentation? 🙏

Reply:
No idea what happened while pushing, done